irohad/main/application.hpp

```diff
@@ -22,22 +22,28 @@
 #include <ametsuchi/ametsuchi.hpp>
 #include <dao/dao.hpp>
 
-namespace main {
+namespace iroha {
 
   /**
    * Contains instances of global services and shared providers
    */
+  using namespace iroha;
   class Irohad {
```

Comment:
What is the role of Irohad?
In other words, what is the difference between instances in this class and `main()`?